packages/eslint-config/eslint-config.cjs

```diff
@@ -0,0 +1,18 @@
+// Used only by external packages.
```

Comment:
I agree that `test262/*` was a non-sequitur, so I've removed it.  Also, I've renamed `plugin:@endo/relaxed` to `plugin:@endo/strict` to better reflect its meaning.

`plugin:@endo/recommended` is just for the specific rules that `@endo/eslint-plugin` defines.  It's a tiny subset of the rules used by `plugin:@endo/internal`.  `plugin:@endo/strict` is a bunch of coding style rules used by Endo packages (and which I'd argue we should someday reconsider for `agoric-sdk` to DRY up the style config).

The unqualified `"@endo"` config is now backwards compatible for external repos that used it directly (such as `Jessie`).  The types linting configuration was a breaking change to that use case, which is why I separated `plugin:@endo/internal` from `"@endo"`.

Reply:
As discussed in a meeting with @turadg, I've removed `packages/eslint-config`, since external users of it will continue to work with an older published version, and it is easy to create external configurations based on `plugin:@endo/strict`.